template/source/layouts/header_footer_only.erb

```diff
@@ -1,91 +0,0 @@
-<!doctype html>
```

Comment:
Except if you removed it, next time you update to the latest template it'd come back again (with 'please remove me').

I tend to just run `diff -qr source ../tech-docs-template/template/source` when I've updated. It's something we could look at improving though (removing old files)